webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -48,6 +48,47 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
         });
     };
 
+    renderRow = (sub: ChannelSubscription): JSX.Element => {
+        const projectKey = sub.filters.projects[0];
+        let projectName = projectKey;
+
+        const project = this.props.jiraProjectMetadata.projects.find((opt) => opt.value === projectKey);
+        if (project) {
+            projectName = project.label;
+        }
+
+        return (
+            <tr
+                key={sub.id}
+                className='select-channel-subscriptions-row'
+            >
+                <td>
+                    <span>{sub.name || '(no name)'}</span>
+                </td>
+                <td>
+                    <span>{projectName}</span>
+                </td>
+                <td>
+                    <button
+                        className='style--none color--link'
```

Comment:
```suggestion
                        className='style--none color--link'
                        style={{marginLeft: '0px'}}
```

Reply:
@mickmister Here's a suggestion.  The buttons are styled with `margin-left: 15px`, which is creating two problems. 1) The shift of the `Edit - Delete` text to the right of the `Actions` header and 2) the uneven centering of the dash between Edit and Delete.